### PR TITLE
Fix build without Arrow

### PR DESCRIPTION
**Pull request by dominiklohmann**
### :notebook_with_decorative_cover: Description

This fixes the build without Arrow and tests it in CI. 

### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

File: vast/integration/vast_integration_suite.yaml
Comment (dominiklohmann) [inline vast/integration/vast_integration_suite.yaml:502]: Should the Zeek tag be Suricata instead?

**Review by dominiklohmann — COMMENTED** [review]
LGTM except for two things: A minor comment, and please add a separate changelog entry considering this was after the rc1 release.

I obviously cannot approve this myself, so consider this approval, approve yourselves and feel free to merge.

**Review by lava — APPROVED** [review]
Approving myself after getting informal approval from @dominiklohmann .

**Pull request merged**
